<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Z3D::ZLTSCameraPreviewer</class>
 <widget class="QMainWindow" name="Z3D::ZLTSCameraPreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTS preview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QPushButton" name="autoSetMinMaxButton">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QDoubleSpinBox" name="rangeMaxSpinBox">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="7">
         <widget class="QDoubleSpinBox" name="rangeMinSpinBox">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="visibleImagesSpinBox">
          <property name="suffix">
           <string> images</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="11">
         <widget class="Z3D::ScanImageViewer" name="scanImageView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="scanViewDisplayModeComboBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QCheckBox" name="scaleValuesCheckBox">
          <property name="text">
           <string>Scale values</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="plotLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlayPause"/>
   <addaction name="separator"/>
   <addaction name="actionAutoSearchROI"/>
   <addaction name="separator"/>
   <addaction name="actionUnsharpMasking"/>
   <addaction name="separator"/>
   <addaction name="actionDisplayCalibratedValues"/>
   <addaction name="separator"/>
   <addaction name="actionShowCameraSettings"/>
  </widget>
  <action name="actionPlayPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play/pause</string>
   </property>
  </action>
  <action name="actionShowCameraSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionAutoSearchROI">
   <property name="text">
    <string>Auto search ROI</string>
   </property>
  </action>
  <action name="actionDisplayCalibratedValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display calibrated values</string>
   </property>
  </action>
  <action name="actionUnsharpMasking">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unsharp masking</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Z3D::ScanImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>zscanimageviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
